.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
```

Comment:
plans to share this are in https://github.com/libero/publisher/issues/375 and hence out of scope for this PR